test/core/TestTsdbQuery.java

```diff
@@ -2466,7 +2466,42 @@ public void runMimMaxFloatOffset() throws Exception {
     }
     assertEquals(600, dps[0].size());
   }
- 
+  
+  @Test
+  public void runPercentiles() throws Exception {
```

Comment:
We need tests for each type of new aggregator as well as making sure the CTors throw the proper exceptions if someone tries to hack the source and construct an aggregator with some funky value like -42.5 as the percentile. Aggregators need more UTs either way :(